features/whitehall.feature

```diff
@@ -100,7 +100,7 @@ Feature: Whitehall
     Given I am benchmarking
     And I am testing through the full stack
     And I force a varnish cache miss
-    When I visit "/government/how-government-works" on the "whitehall-frontend" application
+    When I visit "/government/how-government-works"
```

Comment:
Should we do the same to the "Whitehall offers a world location API" test further down?  Doesn't seem to make much sense that we check that against admin boxes when it's a public API (isn't it?)

Reply:
Yeah, I ignored that as I think it's working (and should generally work), but yes, that test is similarly confusing, so I'm happy to change it as well.

Reply:
Done in 58e3ed1

Reply:
:+1: - I guess the real test is shipping this and seeing that it still passes.